script/main.py

```diff
@@ -23,25 +23,25 @@ def init_argparser():
   return args
 
 
-def find_records(people_json):
-  record_names_phone = []
-  record_names_with_dob =[]
-  dict_json_to_list = people_json['results']
-  for dict_single_record in dict_json_to_list:
-    if 'phone' and 'cell' in dict_single_record:
-      record_names_phone.append(dict_single_record)
-    if 'dob' in dict_single_record:
-      record_names_with_dob.append(dict_single_record)
-  return (record_names_phone, record_names_with_dob)
+def convert_dict_to_list_access_phone_cell_dob_field(people):
```

Comment:
Please make it shorter. Maybe: `extract_phone_numbers` or similar.
And what is `dob`? Maybe it is better to move it to separate function?

Reply:
"dob" is the field in the record the same as "telephone" or "cell"